### DISCUSSION TITLE
Best models for 16GB ram?

### DISCUSSION THREAD

**Discussion by Brog33** [janhq/jan#1348] [General]
Got a M1 Pro Macbook with 16GB ram. Would it completely choke trying to run one of the larger models? (20-40GB). Any experiences?

**Comment by 0xgokuz** [comment]
I think you can just look at the Recommended models 😄

**Reply by Brog33** [reply]
Are those recommendations based off system specs or is it the same for everyone I wonder..

**Comment by ghost** [comment]
You can try any 7B models like Dolphin Mistral, or Solar 10.7B. Some 13b models may struggle with 16GB RAM. And any bigger may actually lead critical system failure. I don't recommend it.
